### DISCUSSION TITLE
Stuttering sound on Wilma

### DISCUSSION THREAD

**Discussion by mortim** [linuxmint/discussions#553] [Tutorials]
> DISCLAIMER : This is not my tutorial, it was written by the discord user **dr.holo** (from the [Linux Mint](https://discord.gg/mint) discord server). I got his permission to repost the tutorial here for more visibility.

> There is the link of the original thread of this tutorial for more information : https://discord.com/channels/628978428019736619/1267060297513041930

HOW TO FIX  THIS  first of all do not fall back to pulseaudio it will be worse!

edit file :
```sudo nano /usr/share/pipewire/pipewire.conf```

default.clock.rate        = 192000
default.clock.max-quantum = 4096
default.clock.quantum     = 2048

remember to remove # before those settings in config

next thing is:
```systemctl edit --user wireplumber.service```

Add this then save the file and exit:

```ExecStartPre=/bin/sleep 1```
then restart everything
```systemctl --user restart pipewire pipewire-pulse wireplumber```

that fixed it for me, i hope it will help someone

**Comment by claudiux** [comment]
I had to restart the computer to get sound after applying these changes.
